store.go

```diff
@@ -211,6 +211,31 @@ func (s *Store) MustFind(q Query) *ResultSet {
 	return rs
 }
 
+// Reload refreshes the record with the data in the database and makes the
+// record writable.
+func (s *Store) Reload(record Record) error {
+	if record.GetID().IsEmpty() {
```

Comment:
ID is already the name of a Model property
https://github.com/src-d/go-kallax/blob/base/model.go